The install-sh scripts chroot bin/attr to root, breaking the sandbox.

diff -ur attr-2.4.46-orig/include/install-sh attr-2.4.46/include/install-sh
--- attr-2.4.46-orig/include/install-sh	2010-01-22 23:55:11.000000000 +0100
+++ attr-2.4.46/include/install-sh	2012-02-19 16:13:47.000000000 +0100
@@ -106,7 +106,7 @@
 
 if $INSTALL
 then
-    CP=cp; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=_chown
+    CP=cp; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=true
 else
     CP=true; LN=true; MKDIR=true; CHMOD=true; CHOWN=true
 fi
diff -ur attr-2.4.46-orig/install-sh attr-2.4.46/install-sh
--- attr-2.4.46-orig/install-sh	2011-04-19 19:55:48.000000000 +0200
+++ attr-2.4.46/install-sh	2012-02-19 16:13:36.000000000 +0100
@@ -106,7 +106,7 @@
 
 if $INSTALL
 then
-    CP=cp; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=_chown
+    CP=cp; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=true
 else
     CP=true; LN=true; MKDIR=true; CHMOD=true; CHOWN=true
 fi
